### ISSUE TITLE
+routes+ error after 0.12.0 -> 0.13.1

### ISSUE THREAD

**Issue by dryewo** [metosin/compojure-api#33]
Hi Tommi,

I've upgraded from `0.12.0` to `0.13.1` (to make use of `{s/Keyword s/Any}`) and started getting this error:
```
Unable to resolve symbol: +routes+ in this context, compiling:(my_project/apis/pizzas.clj:141:13)
```

I have the following structure:
`core.clj`:
```
(ns my-project.core
  (:use clojure.pprint)
  (:require [com.duelinmarkers.ring-request-logging
             :refer [wrap-request-logging]]
            [compojure.api.sweet :refer :all]
            [clj-stacktrace.repl]
            [my-project.apis [pizzas :as pizzas]
              [sushis :as sushis]
              ;; some more imports
              ]
            [my-project.util [middleware :as middleware]]))

(defapi app
        (middlewares [middleware/wrap-catch-exceptions]
                     (swagger-ui)
                     (swagger-docs
                       :title "my project API"
                       :apiVersion "0.4.7"
                       :description "my project description"
                       :termsOfServiceUrl "http://www.google.com"
                       :contact "info@google.com"
                       :license "Eclipse 1.0"
                       :licenseUrl "http://www.eclipse.org/legal/epl-v10.html")
                     pizzas/route
                     sushis/route
                     ;; some more routes
                     )
        (compojure.route/not-found "<h1>No.</h1>"))

(def handler
  (-> #'app
      middleware/wrap-print-request
      (middleware/wrap-access-control-allow "*" "GET, POST, PUT, DELETE")
      ;middleware/wrap-print-response
      ;; Config in src/log4j.properties
      com.duelinmarkers.ring-request-logging/wrap-request-logging
      ))
```

`apis/pizzas.clj`:
```
;; ...

(defmodel Pizza
  ;; ...
  )

(defroutes* route
            (swaggered "pizzas"
                       :description "Pizzas API"
                       (context "/pizzas" []
                                (GET* "/" {pms :params}
                                      :summary "Get all pizzas"
                                      :return [Pizza]
                                      (ok (get-all-pizzas pms))))))
```

I like it because it allows me to isolate api groups from each other.

Now I started investigating and after removing `(swaggered)` from pizzas.clj made it successfully compile.But pizzas disappeared from the swagger front page.

Then I added `(swaggered)` to `core.clj` like this:
```
(swaggered "pizzas"
  :description "Pizzas API"
  pizzas/route)
```
Then it gave an error about "unable to resolve Pizza", because `(:require [my-project.apis [pizzas :as pizzas]])`, not `(:require [my-project.apis.pizzas :refer :all])`.
I tried to `:refer :all`, but it gave errors about coinciding var names (like `route` and `DATE_FORMAT`, that I have in several files).

In fact, my APIs are more complicated than this, they include many schemas which I don't want to explicitly :refer in `core.clj`, and there are many vars with same names across different files.

Is there a possibility in `0.13.1` to define this structure without repeating and adding boilerplate code?

Thanks!

**Comment by ikitommi** [comment]
will be fixed in https://github.com/metosin/compojure-api/tree/swagger2routes

**Comment by ikitommi** [comment]
fix merged to master, will be in `[0.20.0]`

**Comment by dryewo** [comment]
wow thanks
finally we have it :)
